plume-models/src/users.rs

```diff
@@ -320,13 +320,46 @@ impl User {
             .load::<User>(conn)
             .map_err(Error::from)
     }
-
     pub fn outbox(&self, conn: &Connection) -> Result<ActivityStream<OrderedCollection>> {
-        let acts = self.get_activities(conn)?;
-        let n_acts = acts.len();
         let mut coll = OrderedCollection::default();
+        let first = ap_url(&format!("{}?page=1", &self.outbox_url));
```

Comment:
It starts at 1 for users, but 0 for blogs. Either are fine, but it would be better if it was consistent.

Reply:
Looks like it should be one given the current `Page` code.

Let's make it that then.